src/utils/avatar/index.ts

```diff
@@ -0,0 +1,7 @@
+import md5 from 'md5';
+
+export function getAavatarURL(email: string) {
```

Comment:
A letra 'a' em Avaatar está duplicado

```suggestion
export function getAvatarURL(email: string) {
```